main.js

```diff
@@ -22,15 +22,16 @@ function createWindow(on_load) {
     newWindow.loadURL('file://' + __dirname + '/index.html');
 }
 
-function newDocument() {
+function newProject() {
     createWindow(function(win) {
-        win.webContents.send('new-document');
+        win.webContents.send('new-project');
     });
 }
 
-function openDocument(path) {
+function openProject(path) {
     createWindow(function(win) {
-        win.webContents.send('open-document', path);
+        path = path.replace('app.tbinfo','');
```

Comment:
Didnt know the path module. Isnt it kinda weird to have both a module and a variable called "path"?
We only go one level up if the extension is 'tbinfo' cause when he can't find it it wont replace anything. will check the path module when Im home